src/index.js

```diff
@@ -239,9 +248,13 @@ export default class RNPickerSelect extends PureComponent {
 
             // If TextInput is below picker modal, scroll up
             if (textInputBottomY > modalY) {
-                this.props.scrollViewRef.current.scrollTo({
-                    y: textInputBottomY - modalY + this.props.scrollViewContentOffsetY,
-                });
+                // Wait until the modal animation finishes, so the scrolling animation doesn't run at the same time as
```

Comment:
@0xmiroslav @Julesssss

To ensure that the "blank space" doesn't appear, I made scrolling aware of the duration of the modal animation.

This creates some cross-awareness between the scrolling to open picker (which is `react-native-picker-select`-specific) and `PickerAvoidingView`.

Maybe this means that a better target for this functionality in the upstream is actually `react-native-picker-select`?

Reply:
@0xmiroslav @Julesssss 
There a few benefits:
* The `PickerAvoidingView` and the scrolling logic are somewhat aware of each other
* `react-native-picker-select` is our direct picker dependency. It used not to depend on `@react-native-picker/picker` and _theoretically_ it might not depend on it again (although unlikely)
* We already have a fork, so I can just select these commits and submit them as a PR

Unless you oppose, I'll proceed with creating an upstream PR in `react-native-picker-select` 🙃

Reply:
Thanks, yeah I think your suggestion makes sense to me. Better to separate the two libs to keep the implementation simpler.

Reply:
[PR filed](https://github.com/Expensify/react-native-picker-select/pull/11)!

It's stacked on top of one of our PRs, from January. Who knows, maybe the maintiner will wake up 😒

Reply:
We can still review/merge our fork for now. 

Also, I'm trying to offer the maintainer money in order to review our PR. But I'm struggling to find any contact details.

Reply:
@Julesssss That PR was meant for the upstream, but I just figured out that  filed it to our fork 🙈 That's how doing things late in the evening ends.

Reply:
[Upstream PR](https://github.com/lawnstarter/react-native-picker-select/pull/507)

Reply:
@Julesssss 
> Also, I'm trying to offer the maintainer money in order to review our PR. But I'm struggling to find any contact details.

The company which probably owns the GitHub [organization](https://github.com/lawnstarter):
https://www.lawnstarter.com/contact

Author/maintainer who used to work for LawnStarter till 2021:
https://github.com/lfkwtz
https://www.linkedin.com/in/lefkowitzmichael/
lefkowitz.michael (at) gmail.com

Reply:
Ah, thanks. I had reached out on Linkedin but couldn't find their emails anywhere! I just sent a message.